discovery-infra/start_discovery.py

```diff
@@ -369,6 +372,17 @@ def _create_node_details(cluster_name):
     }
 
 
+def _resource_param(value, param_name):
+    for operator in utils.get_env("OLM_OPERATORS", "").lower().split():
```

Comment:
`utils.get_env("OLM_OPERATORS", "").lower().split()` is repeated twice in this PR, maybe we can put it inside a function (`def parse_olm_operators_from_env` or something similiar) so this parsing logic is not repeated